scc/controller.py

```diff
@@ -52,7 +52,7 @@ def get_id(self):
 		return self._id
 
 
-	def get_gui_config_file(self) -> None:
+	def get_gui_config_file(self) -> Optional[str]:
```

Comment:
We don't need Optional, so we can just do `str | None` if needed here, though I don't get what override magic would cause this function to return anything but None, could you explain?